meteomatics/rounding.py

```diff
@@ -1,47 +1,59 @@
 # -*- coding: utf-8 -*-
 
 decimal_places = {
-    "cm":       1,  # centimeters (snow height)
+    "bft":      0,  # Beauforts (wind)
+    "cm":      -1,  # centimeters (snow height, hail)
     "C":        1,  # temperature
     "d":       -1,  # days AND degrees...
     "dn":       5,  # datenums
-    "eur":      1,  # euro
     "F":        1,  # temperature
     "ft":       1,  # feet (ceiling height)
+    "gcm2":     4,  # total column air pollution
     "gm3":      1,  # absolute humidity
     "gdd":      1,  # growind degree days
+    "grainsm3": 1,  # grains per m3 (pollen)
     "hdd":      1,  # heating degree days
     "h":        1,  # hours
     "hft":      1,  # hectofeet
     "hPa":      1,  # pressures
-    "Pa":       1,  # pressures
+    "Hz":       6,  # vorticity
+    "mHz":      3,  # vorticity
+    "uHz":      1,  # vortictiy
+    "Pa":       0,  # pressures
```

Comment:
Why change?

Reply:
I adjusted the values according to to https://gitlab.meteomatics.com/LIB/mm_cpp_lib/-/blob/master/include/Meteomatics_MeteoCacheGlobals.h#L227